website/source/docs/external/lxc.html.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: "docs"
+page_title: "Drivers: LXC"
+sidebar_current: "docs-external-plugins-lxc"
+description: |-
+  The LXC task driver is used to run application containers using LXC.
+---
+
+# LXC Driver
+
+Name: `lxc`
+
+The `lxc` driver provides an interface for using LXC for running application
+containers. You can download the external LXC driver [here][lxc-driver].
+
+~> Nomad 0.9 does not maintain backward compatibility for the external LXC driver plugin when it comes to client configuration syntax. With Nomad 0.9, you must use the new [plugin syntax][plugin]. See [plugin options][plugin-options] below for an example.
```

Comment:
I would suggest being more explicit about what changed and what hasn't: e.g. "LXC client setup changed in 0.9: You must use the [plugin syntax][plugin] and install the external lxc driver prior to upgrading.  Job specification remained the same".